src/index.ts

```diff
@@ -62,6 +62,10 @@ export default class AutoCheckElement extends HTMLElement {
     return link.href
   }
 
+  get csrfField(): string {
+    return this.getAttribute('csrf-field') || 'authenticity_token'
+  }
+
```

Comment:
Can we move this to below the `required` setter at the end of the class? Right now, it's right between the `src` getter and setter, so it makes it hard to scan for specific functions.

Also, adding a setter for the attribute would be great.

Reply:
thanks for review.